### PR TITLE
Bug fix / logging

### PR DESCRIPTION
**Pull request by ericxtang**
Small bug fix in basic_broadcaster

Change logging so by default, we don't print out too much.

### CODE REVIEW

File: vendor/github.com/livepeer/go-livepeer-basicnet/network_node.go [hunk 1]
```diff
@@ -74,7 +74,7 @@ func NewNode(listenPort int, priv crypto.PrivKey, pub crypto.PubKey, f *BasicNot
 	}
 	rHost := rhost.Wrap(basicHost, dht)
 
-	glog.Infof("Created node: %v at %v", peer.IDHexEncode(rHost.ID()), rHost.Addrs())
+	glog.V(2).Infof("Created node: %v at %v", peer.IDHexEncode(rHost.ID()), rHost.Addrs())
```
Comment (yondonfu) [inline vendor/github.com/livepeer/go-livepeer-basicnet/network_node.go:77]: This is mainly for my own knowledge - is the choice of V level for log statements just based on the need to group certain log statements together and it doesn't matter what V level you actually use? Or does each V level actually correspond to some severity level i.e. 2 -> warnings

Comment (ericxtang) [reply vendor/github.com/livepeer/go-livepeer-basicnet/network_node.go:77]: Good point.  I'll document the levels.

**Review by dob — COMMENTED** [review]
I don't really like the use of hardcoded arbitrary integers for the log levels. If nothing else we should set package level constants like loglevel.DEBUG, loglevel.INFO to use instead of the hardcoded ints.

**Comment by ericxtang** [comment]
Closing this PR - it's out of date, and I'll incorporate the suggestions in a future PR.

**Pull request closed without merging**